tools/billing_project.py

```diff
@@ -184,7 +184,7 @@ def _billing_deletion_poller():
 
         return False, status
 
-    poll.poll_predicate("Billing project deletion", 1200, 5, _billing_deletion_poller)
+    poll.poll_predicate("Billing project deletion", 7200, 5, _billing_deletion_poller)
```

Comment:
Will it stop polling sooner if the billing project deletion is in an errored state?